__tests__/app.test.js

```diff
@@ -38,3 +38,23 @@ describe("/api/categories", () => {
     });
   });
 });
+
+describe("/api/users", () => {
+  describe("GET", () => {
+    test("200: Should respond with an array of all users", () => {
+      return request(app)
+        .get("/api/users")
+        .expect(200)
+        .expect("Content-Type", "application/json; charset=utf-8")
+        .then(({ body }) => {
+          expect(Array.isArray(body)).toBe(true);
+          expect(body.length > 0).toBe(true);
```

Comment:
the data should be coming back on the **key** of users which is on the key of body instead of directly on the body

Reply:
✅